.github/CODEOWNERS

```diff
@@ -3,3 +3,5 @@
 ci/helm-chart/ @Matthew-Beckett @alexgorbatchev
 
 docs/install.md @GNUxeava
+
+src/node/i18n/locales/zh-cn.json @zhaozhiming
```

Comment:
Note to self: may have to give write access...

Related: https://github.com/orgs/community/discussions/23042

Reply:
I don't know much about `.github/CODEOWNERS`. Is there anything else I need to do?

Reply:
No, you're good here!